ff-fft/src/lib.rs

```diff
@@ -15,9 +15,11 @@ extern crate alloc;
 
 #[cfg(not(feature = "std"))]
 pub(crate) use alloc::{borrow::Cow, collections::BTreeMap, vec::Vec};
+#[cfg(not(feature = "std"))]
+pub(crate) use core::cmp::min;
 
 #[cfg(feature = "std")]
-pub(crate) use std::{borrow::Cow, collections::BTreeMap, vec::Vec};
+pub(crate) use std::{borrow::Cow, cmp::min, collections::BTreeMap, vec::Vec};
```

Comment:
You can import `core::cmp::min` even when `no_std`, is off, so you don't need to make this change =)